### PR TITLE
Ivanbritvic/fee 655 solana adapter for backend submission

### PR DESCRIPTION
**Pull request by ibrica**

### CODE REVIEW

File: packages/solana-adapter/src/getCancelPaymentSignedTx.ts [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  program: Program<HelioIdl>,
+  req: CancelPaymentRequest
+): Promise<string> => {
+  const [pda] = await PublicKey.findProgramAddress(
+    [req.payment.toBytes()],
+    program.programId
+  );
+  const mint = req.mintAddress!;
+
+  const senderAssociatedTokenAddress = await getAssociatedTokenAddress(
```
Comment (SurenAvagyanG) [inline packages/solana-adapter/src/getCancelPaymentSignedTx.ts:22]: you can name this senderTokenAccount, and then pass to object only value, without key
{
 senderTokenAccount
}
the same about other 4

File: packages/solana-adapter/src/getCreatePaymentSignedTx.ts [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+export const getCreatePaymentSignedTx = async (
+  connection: Connection,
+  wallet: AnchorWallet,
+  program: Program<HelioIdl>,
+  req: CreatePaymentStateRequest,
+  payFees: boolean = true
+): Promise<string> => {
+  const mint = req.mintAddress!;
+
+  const senderAssociatedTokenAddress = await getAssociatedTokenAddress(
```
Comment (SurenAvagyanG) [inline packages/solana-adapter/src/getCreatePaymentSignedTx.ts:29]: same here and other funcions

File: packages/solana-adapter/src/getCreatePaymentSignedTx.ts [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+    helioFeeWalletKey
+  );
+
+  const daoFeeTokenAccountAddress = await getAssociatedTokenAddress(
+    mint,
+    daoFeeWalletKey
+  );
+
+  // eslint-disable-next-line
+  const [_pda, bump] = await PublicKey.findProgramAddress(
```
Comment (SurenAvagyanG) [inline packages/solana-adapter/src/getCreatePaymentSignedTx.ts:55]: const [, bump] - this way you don't have to disable eslint

**Review by SurenAvagyanG — CHANGES_REQUESTED** [review]

File: packages/solana-adapter/src/getCancelPaymentSignedTx.ts (hunk 1 above)
Comment (ibrica) [reply packages/solana-adapter/src/getCancelPaymentSignedTx.ts:22]: Ok

File: packages/solana-adapter/src/getCreatePaymentSignedTx.ts (hunk 2 above)
Comment (ibrica) [reply packages/solana-adapter/src/getCreatePaymentSignedTx.ts:29]: ok

File: packages/solana-adapter/src/getCreatePaymentSignedTx.ts (hunk 3 above)
Comment (ibrica) [reply packages/solana-adapter/src/getCreatePaymentSignedTx.ts:55]: Ok

**Pull request merged**